src/components/SearchResults.js

```diff
@@ -172,6 +172,7 @@ const SearchResults = ( props ) => {
 				noResult={ noResult }
 				postId={ postId }
 				source={ source }
+				showFeedbackSection={ ! resultContent.includes( 'do not possess the answer' ) }
```

Comment:
I think we can maintain a list of phrases we can use here, there might be more ways the bot answers for this. for now we can have this but we should have a way to easily add more items to the list when required.